src/workerd/api/global-scope.h

```diff
@@ -395,6 +396,12 @@ class ServiceWorkerGlobalScope: public WorkerGlobalScope {
     JSG_NESTED_TYPE(WebSocket);
     JSG_NESTED_TYPE(WebSocketPair);
 
+    if (flags.getSqlBinding()) {
+      JSG_NESTED_TYPE(SqlDatabase);
+      JSG_NESTED_TYPE(SqlPreparedStatement);
+      JSG_NESTED_TYPE(SqlResult);
+    }
```

Comment:
Rather than introducing new globals, I would much prefer to see us bring these in as imports using something like...

```
import { SqlDatabase, SqlPreparedStatement, SqlResult } from 'cloudflare:sql';
```

Reply:
Agreed with @jasnell, but also for now I think we can get away with not mapping these types at all, if the SQLite API hangs off of Durable Objects storage.